src/app.js

```diff
@@ -6,8 +6,343 @@ import _ from 'underscore';
+};
+
+// const renderAllTrips = function
 $(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+  tripsTemplate = _.template($('#trips-template').html());
+  tripDescriptionTemplate = _.template($('#trips-description-template').html());
+  reserveFormTemplate = _.template($('#reserve-trip-form-template').html());
+  tripFormTemplate = _.template($('#add-trip-form-template').html());
+  continentTemplate = _.template($('#continent-template').html());
+
+  // tripList.fetch();
+  $('.sort').on('click', sortTrips);
+  console.log(renderReserveForm);
+  // $('#reserve-button').on('click', renderReserveForm);
+  tripList.on('update', renderTrips);
+  tripList.on('sort', renderTrips);
+
+  $('.continent-dropbtn').on('click', function(){
+    $("#continent-dropdown").slideToggle( "slow", function() {
+      // Animation complete.
+    });
+  });// end  function
+
+  $('.cost-dropbtn').on('click', function(){
+    $("#cost-dropdown").slideToggle( "slow", function() {
+      // Animation complete.
+    });
+  });// end  function
+  // $("#continent-dropdown").on('click', 'option', function(){
+  //   $('#trip-table').addClass('hidden');
+  //   $('#trip-table').removeClass('show');
+  //   $('#continent-table').removeClass('hidden');
+  //   $('#continent-table').addClass('show');
+  //   tripList.fetchContinent({
+  //     myValue: $(this).attr('value'),
+  //     success: function(myValue){}
+  //   });
+  // });
+
+  $("#continent-dropdown, #cost-dropdown").on('click', 'option', function(){
+    console.log($(this).attr('id'));
+    let queryUrlSnippet;
+    let parentElement = $(this).parent();
+    if ($(parentElement).attr('id') === 'continent-dropdown'){
+      queryUrlSnippet = 'continent';
+    }else{
+      queryUrlSnippet = 'budget';
+    }
+    // console.log(queryUrlSnippet);
+    console.log('that is queryUrlSnippet')
+    $('#trip-table').addClass('hidden');
+    $('#trip-table').removeClass('show');
+    $('#continent-table').removeClass('hidden');
+    $('#continent-table').addClass('show');
+    tripList.fetchContinent({
+      queryType: queryUrlSnippet,
+      myValue: $(this).attr('value'),
+      success: function(myValue){}
+    });
+  });
+
+  // $("#cost-dropdown").on('click', 'option', function(){
+  //   $('#trip-table').addClass('hidden');
+  //   $('#trip-table').removeClass('show');
+  //   $('#continent-table').removeClass('hidden');
+  //   $('#continent-table').addClass('show');
+  //   tripList.fetchContinent({
+  //     queryType: 'cost',
+  //     myValue: $(this).attr('value'),
+  //     success: function(myValue){}
+  //   });
+  // });
+
+
+  $('#add-trip').on('click', renderTripForm);
+  $('#view-all-trips').on('click', function(){
+    $('#trip-table').addClass('show');
+    $('#trip-table').removeClass('hidden');
+    $('#continent-table').removeClass('show');
+    $('#continent-table').addClass('hidden');
+    tripList.fetch({
+      success: function(collection, response){}
```

Comment:
You should also have an `error` event handler to let the user know when the API call fails.